go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/web-platform-tests/wpt.fyi
 
-go 1.22.7
-
-toolchain go1.22.9
+go 1.23.5
```

Comment:
Any reason to not change this line to:
```suggestion
go 1.23.6
```

Reply:
Somehow I missed this, or messed it up - good catch!